README.md

```diff
@@ -272,6 +272,15 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
+                // The desired method with which to validate OIDC access tokens.
+                // The two methods supported are "local_validation" and
+                // "introspection".
+                //
+                // The following values are supported:
+                // - local_validation: Validate JWT access tokens according to RFC 9086
+                // - introspection: Validate access tokens according to RFC 7662
```

Comment:
Missing period at the end of each item. Adding the periods make it align with documentation for other config options.